tests/models/regression/checkpoint/checkpoint

```diff
@@ -0,0 +1,2 @@
+model_checkpoint_path: "C:\\src\\tensorflow-onnx\\tests\\models\\regression\\checkpoint\\model"
+all_model_checkpoint_paths: "C:\\src\\tensorflow-onnx\\tests\\models\\regression\\checkpoint\\model"
```

Comment:
i know it does not really matter, but just don't feel good about an absolute path.  :-)